src/main/java/pl/tomaszdziurko/jvm_bloggers/view/login/LoginPage.java

```diff
@@ -65,11 +75,11 @@ private void tryToLoginUser(String clientAddress) {
 
         CustomFeedbackPanel feedbackPanel = new CustomFeedbackPanel("feedbackPanel");
         loginForm.add(feedbackPanel);
-        RequiredTextField<String> loginField = new RequiredTextField<>(LOGIN_FIELD_ID);
+        RequiredTextField loginField = new RequiredTextField(LOGIN_FIELD_ID);
```

Comment:
What is the reason of removing type parameter from generic `RequiredTextField`? This introduces compiler warning - please revert.